.travis.yml

```diff
@@ -10,11 +10,7 @@ git:
 
 addons:
   apt:
-    sources:
-      - ubuntu-toolchain-r-test
-      - sourceline: 'ppa:kzemek/boost'
     packages:
-      - g++-6
```

Comment:
Travis comes with GCC 5.4.0 but we use clang anyway so no need to install a different GCC version